src/main/kotlin/no/nav/klage/dokument/service/DokumentUnderArbeidService.kt

```diff
@@ -480,6 +493,11 @@ class DokumentUnderArbeidService(
                 this is JournalfoertDokumentUnderArbeidAsVedlegg
     }
 
+    private fun DokumentUnderArbeid.isHoveddokument(): Boolean {
```

Comment:
Lurer på om du kunne brukt `is DokumentUnderArbeidAsHoveddokument` i stedet?

Reply:
Jovisst! Rettet opp nå.